staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/admission.go

```diff
@@ -171,6 +173,9 @@ func (a *MutatingWebhook) ValidateInitialization() error {
 	if err := a.convertor.Validate(); err != nil {
 		return fmt.Errorf("MutatingWebhook.convertor is not properly setup: %v", err)
 	}
+	if a.defaulter == nil {
+		return fmt.Errorf("MutatingWebhook.defaulter is not properly setup: %v")
```

Comment:
What is the %v supposed to resolve to?

Reply:
I think it will say "%!v(MISSING)"

Reply:
Thanks. Sent https://github.com/kubernetes/kubernetes/pull/58187.